docs/docs/config/system.md

```diff
@@ -135,3 +135,40 @@ Following [split keyboard](../features/split-keyboards.md) settings are defined
 | `CONFIG_ZMK_SPLIT_BLE_PERIPHERAL_STACK_SIZE`            | int  | Stack size of the BLE split peripheral notify thread                       | 650                                        |
 | `CONFIG_ZMK_SPLIT_BLE_PERIPHERAL_PRIORITY`              | int  | Priority of the BLE split peripheral notify thread                         | 5                                          |
 | `CONFIG_ZMK_SPLIT_BLE_PERIPHERAL_POSITION_QUEUE_SIZE`   | int  | Max number of key state events to queue to send to the central             | 10                                         |
+
+## Snippets
+
+Snippets are a way to save common configuration separately when it applies to multiple different applications.
+
+More documentation is avaialable from [Zephyr](https://docs.zephyrproject.org/latest/build/snippets/index.html).
+
+ZMK implements the following snippets:
```

Comment:
```suggestion
```
I wouldn't consider this to be a general "Snippets" section, rather just "These are the snippets that fall under system configuration".

Reply:
The issue here is there is no general "Snippets" section, this should be moved entirely in the future.

Reply:
I'd say "Snippets" is fine, but I'd also be happy with "Memory Configuration".

Reply:
I updated the line to `ZMK implements the following system configuration snippets:` instead, this should hopefully remove the implication this is the only snippet ZMK adds.